### PR TITLE
chore(swap): Adjust ccm type in status v2

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: packages/sdk/src/swap/v2/types.ts [hunk 1]
```diff
@@ -8,11 +8,9 @@ interface DcaParameters {
   chunkIntervalBlocks: string;
 }
 
-interface Ccm {
+interface CcmParameters {
   message: string;
   gasBudget: string;
-  cfParameters: string | undefined;
-  ccmDepositReceivedBlockIndex: string | undefined;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:15]: i think is no separate ccm deposit event after the events refactor

File: packages/sdk/src/swap/v2/types.ts [hunk 2]
```diff
@@ -8,11 +8,9 @@ interface DcaParameters {
   chunkIntervalBlocks: string;
 }
 
-interface Ccm {
+interface CcmParameters {
   message: string;
   gasBudget: string;
-  cfParameters: string | undefined;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:14]: afaik these are not used yet, we can still add them once there is a reason

File: packages/sdk/src/swap/v2/types.ts [hunk 3]
```diff
@@ -101,7 +99,7 @@ interface EgressFields {
 interface SwapStatusResponseCommonFields extends ChainsAndAssets {
   swapId: string;
   destAddress: string;
-  ccm: Ccm | undefined;
+  ccmParams: CcmParameters | undefined;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:102]: makes the property name consistent with `requestDepositAddress`

File: packages/sdk/src/swap/v2/types.ts [hunk 4]
```diff
@@ -114,10 +112,7 @@ interface ReceivingVaultSwap extends VaultSwapCommonFields {
 }
 
 interface SwappingVaultSwap extends ReceivingVaultSwap {
-  swap:
-    | Exclude<SwapChunk, 'latestSwapRescheduledAt' | 'latestSwapRescheduledBlockIndex'>
-    | undefined;
-  swapEgress: EgressFields | undefined;
+  swap: SwapChunk | undefined;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:115]: contract swaps can be rescheduled if the global price impact protection hits

Comment (niklasnatter) [reply packages/sdk/src/swap/v2/types.ts:115]: not sure if we gain anything from separate contract swaps from channel swaps in the types now, but can revisit that later

File: packages/sdk/src/swap/v2/types.ts [hunk 5]
```diff
@@ -133,11 +128,11 @@ interface Receiving extends DepositChannel {
 }
 
 interface Swapping extends Receiving {
-  swap: SwapFields;
+  swap: SwapFields | SwapChunk;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:131]: channel swaps will return `SwapChunk` if the swap is not a dca swap atm

File: packages/sdk/src/swap/v2/types.ts [hunk 6]
```diff
@@ -81,7 +79,6 @@ interface SwapFields {
   isDca: true;
   lastExecutedChunk: SwapChunk | undefined;
   currentChunk: SwapChunk;
-  type: SwapType;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:84]: never set, but also think its fine to not expose this

File: packages/sdk/src/swap/v2/types.ts [hunk 7]
```diff
@@ -92,7 +89,6 @@ interface EgressFields {
   txRef: string | undefined;
   confirmedAt: number | undefined;
   confirmedBlockIndex: string | undefined;
-  ignoredAmount: string | undefined;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:95]: use `amount` and `failure` if egress was ignored (consistent with deposit)

**Review by acdibble — APPROVED** [review]

**Review by jerryafr — APPROVED** [review]

**Pull request merged**